legend-sdlc-protocol-pure/src/main/java/org/finos/legend/sdlc/protocol/pure/v1/PureToEntityConverter.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.protocol.pure.v1;
+
+import org.eclipse.collections.api.factory.Sets;
+import org.eclipse.collections.api.set.ImmutableSet;
+import org.finos.legend.engine.protocol.pure.v1.ProtocolToClassifierPathLoader;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.PackageableElement;
+import org.finos.legend.engine.shared.core.ObjectMapperFactory;
+import org.finos.legend.sdlc.protocol.ProtocolToEntityConverter;
+
+import java.util.Map;
+
+public class PureToEntityConverter extends ProtocolToEntityConverter<PackageableElement>
+{
+    private final Map<Class<? extends PackageableElement>, String> CLASS_TO_CLASSIFIER = ProtocolToClassifierPathLoader.getProtocolClassToClassifierMap();
+
+    private final ImmutableSet<String> SUPPORTED_CLASSIFIERS = Sets.immutable.withAll(CLASS_TO_CLASSIFIER.values());
```

Comment:
SUPPORTED_CLASSIFIERS isn't used, so get rid of it. CLASS_TO_CLASSIFIER isn't static, so use camel case instead of caps and underscores.